neo-gui/UI/ClaimForm.Designer.cs

```diff
@@ -67,20 +69,35 @@ private void InitializeComponent()
             this.textBox2.Name = "textBox2";
             this.textBox2.ReadOnly = true;
             // 
+            // combo_address
+            // 
+            this.combo_address.FormattingEnabled = true;
+            resources.ApplyResources(this.combo_address, "combo_address");
+            this.combo_address.Name = "combo_address";
```

Comment:
```suggestion
            this.combo_address.Name = "combo_address";
            this.combo_address.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
```

Reply:
We should have further discussion about that.

If we change the ComboBoxStyle to DropDownList
`this.combo_address.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;`

Claim gas can not input manually.

In some cases:
Users would like to claim gas to cold wallet.
they have to make one more transaction.

By the way, there is "scripthash" check for manually input address. prevent safe problem.

How are about your suggestion?
Regards.

Reply:
you can add your cold address in your wallet, what do you think?